Changes

```diff
@@ -6,6 +6,8 @@ OCaml 5.2 maintenance version
   if the OCaml stack is expanded during a callback.
   (Miod Vallat, report by Vesa Karvonen, review by Gabriel Scherer and
    Xavier Leroy)
+- #13234, #13267: fix a compatibility issue with Debian armel (armv5) systems
```

Comment:
Please put an empty line above your entry.

Also, "fix a compatibility issue" is too vague IMHO. What about something such as "open runtime events file in read-write mode on armel (armv5) systems due to atomic operations limitations on that platform".

Reply:
Done.